types/local-storage/local-storage-tests.ts

```diff
@@ -0,0 +1,13 @@
+import * as ls from 'local-storage';
```

Comment:
Namespaces are not callable, instead use `import name = require='module'`

Reply:
My bad